pan_indent_checker.py

```diff
@@ -44,9 +48,20 @@ def supports_color():
     return True
 
 
-def _print_debug(k, v):
-    print(DEBUG_DIVIDER)
-    print(DEBUG_LINE % (k, v))
+def _print_debug(k, v, c=""):
+    print(DEBUG_LINE % (k, c[:3], v))
+
+
+def find_annotation_blocks(text):
+    """Find multi-line annotation blocks in a block of text"""
+    result = []
```

Comment:
Personal preference would be to use the plural for a list.

Reply:
Meh.

Reply:
Humph.

Reply:
Put another way, this function was copied from Panlint and I'd like it to stay the same.